src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -1,5 +1,7 @@
 package teammates.test.cases.datatransfer;
 
+import static org.hamcrest.CoreMatchers.not;
+import static org.junit.Assert.assertThat;
```

Comment:
We shall not depend on the dependencies that are not declared explicitly in `gradlew` build file.